week-3/slideshow/index.html

```diff
@@ -3,12 +3,19 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>Image carousel</title>
```

Comment:
Some of the images look stretched. How can you modify the css so that the images are not distorted and retain their actual aspect ratios?

Reply:
by max-width:  and max-height: